docs/api.md

```diff
@@ -1040,6 +1044,14 @@ to use the API after this request has succeeded.
   <!--end-request-body-post-accountdevicedestroy-id-->
 
 
+#### GET /recovery_email/enabled
+
+:lock: HAWK-authenticated with session token
+<!--begin-route-get-recovery_emailenabled-->
+Returns whether or not secondary emails is enabled for a user.
+<!--end-route-get-recovery_emailenabled-->
```

Comment:
Going to update the name to be very specific in the hope that we eventually remove it and have better feature state handling between auth and content servers.